libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -404,3 +404,19 @@ export const firetextConfig: IConfigCredentials[] = [
   },
   ...smsConfigBase,
 ];
+
+export const outlook365Config: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.User,
+    displayName: 'User',
+    description: 'Full email address. (e.g. jdoe@mycompany.com)',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Password,
+    displayName: 'Password',
+    type: 'string',
+    required: true,
+  },
+];
```

Comment:
I'd suggest to reuse the `mailConfigBase` and map the `User` property to the `From` property, then add the credential password.